### ISSUE TITLE
Bug: Add to Remindoro create just empty remindoros 

### ISSUE THREAD

**Issue by Jackymancs4** [palerdot/remindoro#3]
The `Add to Remindoro` context menu item simply creates an empty Remindoro when clicking both on a link or a page.

It should create a remindoro with at least the url in the description.

**Comment by Jackymancs4** [comment]
After testing on Chrome I see that both this and issue #2 works as intended.
On Firefox the extension fail with
```
ReferenceError: to_add is not defined events-modular.js:139:10
```

This might be due to the Firefox manifest missing the [activeTab](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#activeTab_permission)

**Comment by palerdot** [comment]
Thanks for raising this issue ... will check why this is failing in firefox ...

**Comment by palerdot** [comment]
This issue is fixed in latest `0.2.3` version of both Firefox and Chrome. Thank you for bringing this to notice ...
